### PR TITLE
Package property sizing fixes

### PR DESCRIPTION
**Pull request by etbyrd**
Simple fix for: https://github.com/dotnet/project-system/issues/4632

### CODE REVIEW

File: src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx [hunk 1]
```diff
@@ -1828,7 +1828,7 @@
     <value>22</value>
   </data>
   <data name="TableLayoutPanel.Size" type="System.Drawing.Size, System.Drawing">
-    <value>2011, 917</value>
+    <value>1160, 900</value>
```
Comment (Pilchie) [inline src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx]: This doesn't seem right to me.  Shouldn't we be using auto-sizing?  Something like setting `DockMode` to `Dock.Fill`?

Comment (drewnoakes) [reply src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx]: Agree. @etbyrd let's have a look at this one together.

**Comment by etbyrd** [comment]
Added some changes to fix not just the size of the page, but the formatting problem with the assembly version boxes.

<details>
  <summary>Whole page showing where the scroll bar starts now and assembly version fixes</summary>
  image1
</details>

<details>
  <summary>Approximately same content size with both horizontal and vertical scroll bars visible and nonuniform gap between assembly version boxes </summary>
 image2
</details>

File: src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx [hunk 2]
```diff
@@ -705,14 +705,17 @@
   <data name="&gt;&gt;LicenseLineLabel.ZOrder" xml:space="preserve">
     <value>17</value>
   </data>
+  <data name="LicenseUrlWarning.Dock" type="System.Windows.Forms.DockStyle, System.Windows.Forms">
+    <value>Fill</value>
+  </data>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx]: Is this necessary?

File: src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx (hunk 1 above)
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.resx]: @Pilchie I believe this has been addressed now.

**Pull request merged**